cypress/fixtures/data.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
Would it make sense to isolate test data and configuration-specific data?

Reply:
Separated test data.